st-util/src/main/java/de/saring/util/unitcalc/FormatUtils.java

```diff
@@ -452,7 +452,18 @@ public String heightToString(int height) {
      */
     public String cadenceToString(int cadence) {
         numberFormat.setMaximumFractionDigits(0);
-        return numberFormat.format(cadence) + " rpm";
+        return numberFormat.format(cadence) + " r|spm";
```

Comment:
The unit name "r|spm" is very geeky - every programmer knows the or-sign. But probably not the typical user. I would prefer the unit name "rpm / spm". Also in the diagram axis labels above.

Reply:
Done.